docs/conf.py

```diff
@@ -63,7 +63,7 @@
 # built documents.
 #
 # The full version, including alpha/beta/rc tags.
-release = '0.6.1.dev0'
+release = '0.6.0'
```

Comment:
is this a mistake?

Reply:
Yes. The nifake version wasn't updated so depending on what built this may be the new or the old version. There is a change in another PR that fixes this but I also made the change here.